esp-radio-preempt-baremetal/src/timer/mod.rs

```diff
@@ -56,6 +61,10 @@ extern "C" fn timer_tick_handler(context: &mut TrapFrame) {
     if cfg!(esp32) {
         yield_task();
     } else {
-        crate::task::task_switch(context);
+        #[cfg(target_arch = "xtensa")]
+        crate::task::task_switch(_context);
+
+        #[cfg(target_arch = "riscv32")]
+        crate::task::task_switch();
```

Comment:
This doesn't need to be cfg'd, you could just ignore the argument in the riscv task_switch.